### PR TITLE
Improved AutoCompletion & Fixed Friend Alias bug

### PR DESCRIPTION
**Pull request by derech1e**
- added text hint to chat
closes CCBlueX/LiquidBounce1.8-Issues#3796
- fixed a bug, where the autocomplete start not with the first index
fixes CCBlueX/LiquidBounce1.8-Issues#3795

### CODE REVIEW

File: 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java [hunk 1]
```diff
@@ -48,7 +49,10 @@ private void init(CallbackInfo callbackInfo) {
 
     @Inject(method = "keyTyped", at = @At("RETURN"))
     private void updateLength(CallbackInfo callbackInfo) {
-        if (inputField.getText().startsWith(String.valueOf(LiquidBounce.commandManager.getPrefix())) && !inputField.getText().startsWith(LiquidBounce.commandManager.getPrefix() + "lc"))
+        if (!inputField.getText().startsWith(String.valueOf(LiquidBounce.commandManager.getPrefix()))) return;
+        LiquidBounce.commandManager.autoComplete(inputField.getText());
+
+        if (!inputField.getText().startsWith(LiquidBounce.commandManager.getPrefix() + "lc"))
```
Comment (superblaubeere27) [inline 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java:55]: What was fixed here?

File: 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java [hunk 2]
```diff
@@ -95,6 +99,17 @@ private void handleClientCommandCompletion(String full, final String ignored, Ca
         }
     }
 
+    /**
+     * Add this callback, to check if the User complete a Playername or a Liquidbounce command.
+     * Bug report & more info, see: https://github.com/CCBlueX/LiquidBounce1.8-Issues/issues/3795
```
Comment (superblaubeere27) [inline 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java:104]: Commit messages don't go in comments

File: 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java (hunk 1 above)
Comment (derech1e) [reply 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java:55]: https://github.com/CCBlueX/LiquidBounce1.8-Issues/issues/3795
Watch the Video. In the current build the completion will write the second command suggestion, not the first one.

File: 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java (hunk 2 above)
Comment (derech1e) [reply 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java:104]: Okay, should i remove the whole comment or only the commit url?

Comment (kawaiinekololis) [reply 1.8.9-Forge/src/main/java/net/ccbluex/liquidbounce/injection/forge/mixins/gui/MixinGuiChat.java:104]: I think this comment is fine.

**Pull request merged**